pkg/gcrane/copy_test.go

```diff
@@ -175,26 +175,27 @@ func TestCopy(t *testing.T) {
 	defer s.Close()
 
 	// Make sure we don't actually talk to XCR.
-	http.DefaultTransport = s.Client().Transport
+	opt := remote.WithTransport(s.Client().Transport)
```

Comment:
Rather than take steps to not accidentally talk to xcr.io, how about using registry.example.com which will definitely never exist.

Reply:
Ok this is done.